### ISSUE TITLE
monoallelic_X_heterozygous    

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7495]
**Preferred term label:**
monoallelic_X_heterozygous    

**Synonyms**
X-linked Dominant inheritance
HP:0001423

**Definition (free text, please give PubMed ID)**
The allelic requirement for X-linked Dominant inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as allelic requirement synonyms
